js/containers/portal-dashboard/portal-dashboard-app.tsx

```diff
@@ -204,7 +197,7 @@ class PortalDashboardApp extends React.PureComponent<IProps, IState> {
 
   private handleScroll = (e: React.UIEvent<HTMLElement>) => {
     const element = e.target as HTMLElement;
-    if (element && element.scrollLeft) {
+    if (element && element.scrollLeft !== null) {
```

Comment:
```suggestion
    if (element && element.scrollLeft != null) {
```
Presumably, you want to catch undefined as well.

Reply:
Fixed